pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -104,34 +104,34 @@ func (m *volumeIDToPvcMap) get(volumeHandle string) string {
 	return m.items[volumeHandle]
 }
 
-// Map of volumeID to the list of node names in the cluster.
-// Key is the volume ID and value is the list of published nodes for the volume
+// Map of volumeName to the list of node names in the cluster.
+// Key is the volume name and value is the list of published nodes for the volume
 // The methods to add, remove and get entries from the map in a threadsafe
 // manner are defined.
-type volumeIDToNodesMap struct {
+type volumeNameToNodesMap struct {
 	*sync.RWMutex
 	items map[string][]string
 }
 
-// Adds an entry to volumeIDToNodesMap in a thread safe manner.
-func (m *volumeIDToNodesMap) add(volumeID string, nodes []string) {
+// Adds an entry to volumeNameToNodesMap in a thread safe manner.
+func (m *volumeNameToNodesMap) add(volumeName string, nodes []string) {
```

Comment:
Do we have a need to assign one node at a time to this map.
For example, add one node now and then another node later.
For RWM volume, we might create one Pod with RWM claim, and later create another Pod with the same claim.

Reply:
The reason I am doing it one at a time is because, in the volumeAttachment spec, the `nodeName` and `source.persistentVolumeName` are strings and not a slice. I thought there will be multiple volumeAttachments that will be created when we have 1 volume mapped to multiple node names. So the handler function will be called one at a time in such a scenario. This is the reason I am removing it 1 at a time.